static/survey-ad.less

```diff
@@ -0,0 +1,42 @@
+survey-ad {
+  background: white;
+  border-top: 1px solid @border-color;
+  bottom: 0;
+  position: absolute;
+  text-align: center;
+  transition: bottom @transition-speed ease;
+  width: 100%;
+  z-index: 90;
```

Comment:
Is `90` really needed? I don't doubt it, just curious.

Reply:
Yeah, this is the same `z-index` as '#right`; if I change the code I can update it with a note about where this value comes from.